packages/client/components/jobs/JobCard.tsx

```diff
@@ -1,138 +1,130 @@
 import {
   Box,
-  Container,
-  Link,
   Flex,
-  Spacer,
-  Image,
   Heading,
-  Text,
   HStack,
+  Image,
+  Link,
+  Text,
   Tag,
   TagLabel,
+  VStack,
 } from '@chakra-ui/react';
+import { Job } from '../../types/job'
+import { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHeart as faHeartSolid } from '@fortawesome/free-solid-svg-icons';
+import { faHeart as faHeartOpen } from '@fortawesome/free-regular-svg-icons';
+
+export default function JobCard(props: Job) {
+  // Defaults until we actually get some real data
+  // Probably not entirely necessary but will definitely be destructuring the data in some capacity.
+  const {
+    position = "Frontend Magician",
+    compensation = "0.6 ETH",
+    description = "We already have the best design skills in web3. No chance you’ll be able to improve on what we’ve got but why not come along and learn something?",
+  } = props;
+
+  const name = "Company Name",
+    organization_description = "NFT Marketplace",
+    logo_url = "/DevDAO.png",
+    technologyTags = ["Solidity", "EthersJS"];
```

Comment:
As I wasn't entirely sure how the queries are working as of yet, I opted to only pass in the job specific data as props. This can and most likely will be updated at some point once I have a better understanding of it.